assets/src/dashboard/app/api/useApiAlerts.js

```diff
@@ -19,30 +19,36 @@
  */
 import { useEffect } from 'react';
 import { useSnackbar } from '@web-stories-wp/design-system';
-
+import { useDebouncedCallback } from 'use-debounce';
 /**
  * Internal dependencies
  */
+import { SUCCESS } from '../textContent';
 import useApi from './useApi';
 
 function useApiAlerts() {
-  const { storyError, templateError, settingsError, mediaError } = useApi(
-    ({
-      state: {
-        stories: { error: storyError },
-        templates: { error: templateError },
-        settings: { error: settingsError },
-        media: { error: mediaError },
-      },
-    }) => ({
-      storyError,
-      templateError,
-      settingsError,
-      mediaError,
-    })
-  );
+  const { storyError, templateError, settingsError, mediaError, settingSaved } =
+    useApi(
+      ({
+        state: {
+          stories: { error: storyError },
+          templates: { error: templateError },
+          settings: { error: settingsError, settingSaved },
+          media: { error: mediaError },
+        },
+      }) => ({
+        storyError,
+        templateError,
+        settingsError,
+        mediaError,
+        settingSaved,
+      })
+    );
   const { showSnackbar } = useSnackbar();
 
+  const debouncedShowSnackbar = useDebouncedCallback((message) => {
```

Comment:
could use this for errors as well but didn't really see the need. open to opinions.

Reply:
I think it would be good to use for errors too - especially if no errors are shown after these requests fail.